components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManager.java

```diff
@@ -192,4 +192,13 @@ Organization updateOrganization(String organizationId, String currentOrganizatio
      */
     List<Organization> getOrganizationsByName(String organizationName) throws OrganizationManagementException;
 
+    /**
+     * Return whether the given organization is an immediate child of the super organization.
+     *
+     * @param organizationId Checking organization id.
+     * @return True if the given organization is an immediate child of super organization.
+     * @throws OrganizationManagementServerException Error occurred while checking whether the organization is an
+     *                                               immediate child of the super organization.
+     */
+    boolean isImmediateChildOfSuperOrganization(String organizationId) throws OrganizationManagementServerException;
```

Comment:
Instead of this method, shall we have a method to get the depth of the organization in the hierarchy?

Reply:
fixed in [637af83](https://github.com/wso2-extensions/identity-organization-management-core/pull/8/commits/637af83f3aeb1c008b7cf663c2e1b3f5b2361b08)